src/esp/bindings_js/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ set(
   ${SCENE_DATASETS}/habitat-test-scenes/van-gogh-room.navmesh
   ${MAGNUM_WINDOWLESSEMSCRIPTENAPPLICATION_JS}
   ${MAGNUM_WEBAPPLICATION_CSS}
+  ${DATA_DIR}
```

Comment:
This creates a symlink to `data` so that all your local data files become available when choosing files to preload in index.js.